src/topics/inputs.md

```diff
@@ -212,7 +222,7 @@ parameters together to describe these two conditions.
 :emphasize-lines: 6-7
 ```
 
-In the first example, you can't provide `itemA` without also providing `itemB`.
+In the first example, you can't provide `itemA` without also providing `itemB`, or an error is thrown. This is because they are inclusive (dependent) parameters. If you provide one, you must provide the other.
```

Comment:
Probably better to break this line to make it shorter like the other ones (regardless of `80` or `120` limits). 

This is useful to visualize diffs with git or GitHub UI: https://sembr.org/